allennlp/commands/__init__.py

```diff
@@ -3,35 +3,39 @@
 import logging
 import sys
 
-from allennlp.commands.serve import Serve
-from allennlp.commands.predict import Predict
-from allennlp.commands.train import Train
+from allennlp.commands.elmo import Elmo
 from allennlp.commands.evaluate import Evaluate
+from allennlp.commands.fine_tune import FineTune
 from allennlp.commands.make_vocab import MakeVocab
-from allennlp.commands.elmo import Elmo
+from allennlp.commands.predict import Predict
+from allennlp.commands.serve import Serve
 from allennlp.commands.subcommand import Subcommand
+from allennlp.commands.train import Train
 from allennlp.service.predictors import DemoModel
 
 logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
 
+
 def main(prog: str = None,
          model_overrides: Dict[str, DemoModel] = {},
          predictor_overrides: Dict[str, str] = {},
          subcommand_overrides: Dict[str, Subcommand] = {}) -> None:
     """
-    The :mod:`~allennlp.run` command only knows about the registered classes
-    in the ``allennlp`` codebase. In particular, once you start creating your own
-    ``Model`` s and so forth, it won't work for them. However, ``allennlp.run`` is
-    simply a wrapper around this function. To use the command line interface with your
-    own custom classes, just create your own script that imports all of the classes you want
-    and then calls ``main()``.
-
-    The default models for ``serve`` and the default predictors for ``predict`` are
-    defined above. If you'd like to add more or use different ones, the
-    ``model_overrides`` and ``predictor_overrides`` arguments will take precedence over the defaults.
+    The :mod:`~allennlp.run` command only knows about the registered classes in the ``allennlp``
+    codebase. In particular, once you start creating your own ``Model`` s and so forth, it won't
+    work for them, unless you use the ``--include-package`` flag available for most commands.
+
+    The default models for ``serve`` and the default predictors for ``predict`` are defined above.
+    If you'd like to add more or use different ones, the ``model_overrides`` and
+    ``predictor_overrides`` arguments will take precedence over the defaults.
```

Comment:
this is no longer true, they're defined elsewhere.

also, it occurs to me that now that `predict` has a `--predictor` option there's not really any need for predictor overrides, we should get rid of it

Reply:
Yeah, you're right.  I added a TODO to take care of the argument fixes later - I think we can probably also remove the `model_overrides`, and maybe just replace the `serve` command with the simple server.